tgui-next: Gravity Generator

Arkatos1 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

This PR converts gravity generator to tgui-next UI with some tweaks regarding text and overall structure.

**BEFORE:**

![BeforeGravGen](https://user-images.githubusercontent.com/43862960/69916359-80477400-145a-11ea-8ab9-2ccb5b45adc7.png)

**AFTER: (UPDATED!)**

![FixedGravGen](https://user-images.githubusercontent.com/43862960/69920236-9c143f80-1485-11ea-8293-b5bb41ac15ca.png)

## Why It's Good For The Game

Better and more responsive UI.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl: Arkatos
add: Gravity Generator now uses tgui-next UI.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Arkatos1 committed:
Basic ui procs

Arkatos1 committed:
Basic tgui-next

Arkatos1 committed:
tgui-next gravity generator

stylemistake commented:
> Center aligned section titles.

Stop it. Center alignment is a bad typographical manner, and I have never intended for section titles to be center-alignable.

Section title also serves no purpose, so why not simply remove it and make this UI even more compact. 👍 

> Red colored warning text

This color combination is unreadable, perhaps you wanted to make the whole noticebox red, but it's not doable at the moment (I might add color support in the future). Keep it simple.

stylemistake reviewed (changes requested):
> Center aligned section titles.

Center alignment is a bad typographical manner, and I have never intended for section titles to be center-alignable. This happens way too frequently (medical kiosk is another example of unnecessary center alignment), so I might add a style guide someday.

Section title also serves no purpose, so I'd just say simply remove it and make this UI even more compact. 👍 

> Red colored warning text

This color combination is unreadable, perhaps you wanted to make the whole noticebox red, but it's not doable at the moment (I might add color support in the future). Keep it simple.

**P.S.** Overall, code looks good. 👍

Arkatos1 committed:
Review fixes

Arkatos1 commented:
I have removed section title and adressed red warning color per review. Updated UI image:

![FixedGravGen](https://user-images.githubusercontent.com/43862960/69920162-c9142280-1484-11ea-802d-27cf30b5ecd2.png)

Arkatos1 committed:
Code cleanup

actioninja committed:
rebuild
